IHP/IDE/SchemaDesigner/View/Layout.hs

```diff
@@ -76,10 +76,10 @@ databaseControls = [hsx|
 
 findStatementByName statementName statements = find pred statements
     where
-        pred CreateTable { name } | (toUpper name) == (toUpper statementName) = True
-        pred CreateTable { name } | (toUpper name) == (toUpper (tshow statementName)) = True
-        pred CreateEnumType { name } | (toUpper name) == (toUpper statementName) = True
-        pred CreateEnumType { name } | (toUpper name) == (toUpper (tshow statementName)) = True
+        pred CreateTable { name } | (compileIdentifier name) == (compileIdentifier statementName) = True
+        pred CreateTable { name } | (parseKeyword name) == (parseKeyword statementName) = True
+        pred CreateEnumType { name } | (compileIdentifier name) == (compileIdentifier statementName) = True
+        pred CreateEnumType { name } | (parseKeyword name) == (parseKeyword statementName) = True
```

Comment:
Why these changes?

Reply:
This was necessary when the escaping was done in the parser. I'll revert this.